src/config.js

```diff
@@ -3,11 +3,13 @@ module.exports = {
     lang: process.env.BOT_LANG || 'en',
     port: process.env.PORT || process.env.VCAP_APP_PORT || 3000,
     mongoURI: process.env.MONGODB_URI,
-    admin: process.env.ADMINISTRATOR_NICKNAME || 'dcversus',
+    adminDiscordId: process.env.ADMIN_DISCORD_ID || '258540630681190402',
     selfName: process.env.SELF_NAME || 'ботий',
+    url: process.env.URL || 'http://u.dcvs.ru',
     discord: {
         color: '0xf9690e',
         authToken: process.env.DISCORD_AUTH_TOKEN,
+        broadcastChannelName: process.env.BROADCAST_CHANNEL_NAME,
```

Comment:
its seems like we can get merge conflicts

Reply:
with?